spec/cases/each_with_ar_sqlite.rb

```diff
@@ -29,11 +27,19 @@ class User < ActiveRecord::Base
 print "Parent: "
 puts User.first.name
 
+# https://www.sqlite.org/faq.html
+# Under Unix, you should not carry an open SQLite database across a fork() system call into the child process.
+ActiveRecord::Base.connection.disconnect!
+
 print "Parallel (#{in_worker_type}): "
 Parallel.each([1], in_worker_type => 1) do
+  ActiveRecord::Base.establish_connection(params)
```

Comment:
not working for 0 is fine, in that case we will not execute the threading/forking anyway ... so pick what is simpler even if it does not work for 0